recipes/qdbm/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS KEEP_RPATHS)
+
+add_executable(test_package test_package.c)
+target_link_libraries(test_package PRIVATE CONAN_PKG::qdbm)
```

We prefer to use `cmake_find_package[_multi]` generator, as it is more aligned with the future `CMakeDeps` in Conan v2.

```suggestion
find_package(qdbm REQUIRED)

add_executable(test_package test_package.c)
target_link_libraries(test_package PRIVATE qdbm::qdbm)
```

---

That would require adding "unofficial" CMake targets. Feedback from @SpaceIm was that this is not desirable.

---

conan fallbacks to `<recipeName>Config.cmake` and `recipeName::recipeName` imported targets in these generators. What I suggest is to not enforce an unofficial name in `package_info()`, but you can use `cmake_find_package_multi` in test package (there will be no other choice in conan v2 anyway).